### DISCUSSION TITLE
申请退款！！！！！

### DISCUSSION THREAD

**Discussion by jinghanoa** [getlantern/lantern-forum-cn#1024] [General]
太慢了太慢了
蓝灯邮箱：alexalele@163.com
支付时间
2024-11-06 14:47:35
付款方式
工商银行储蓄卡

游戏充值订单号P01202411061447023512362

收款方全称
广州饶铭网络科技有限公司
订单号
2024110622001456111408780071
商家订单号
2024110614470571193

**Comment by MaxMKMax** [comment]
@jinghanoa 您好. 已经帮您提交问题上去了. 请您耐心等待. 谢谢.

**Reply by jinghanoa** [reply]
给我个时限

**Reply by alexa1111ee** [reply]
我也没有收到退款。。为什么我自动登录后显示不是专业版了。。。还给我限流了？？？能不能解决？？？

**Reply by MaxMKMax** [reply]
@alexa1111ee 您好. 应该是系统问题. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去.

同时您也查看您是否是最新版本? 更新后然后重新登入应该可以了.

**Comment by MaxMKMax** [comment]
@jinghanoa 您好. 退款不是我处理的, 我只是中间人传话, 我无法给您提供时限. 问题已经帮您提交上去了. 但是一般整个过程完成退款要 3 周到 4 周.
